config/config-api/src/main/java/com/thoughtworks/go/config/MailHost.java

```diff
@@ -181,6 +181,8 @@ public void setAdminMail(String adminMail) {
 
     @PostConstruct
     public void ensureEncrypted() {
+        this.username = StringUtils.stripToNull(username);
```

Comment:
Rename the method as we are not just dealing with encryption at post-construct time?

Reply:
It's part of the interface, so unfortunately, I cannot.